static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -50,8 +49,11 @@ define([
             new MtStickyNav(),
             new AcrossTheCountry(),
             new AdblockMessage(),
-            new MtStickyBottom()
-        ],
+            new MtStickyBottom(),
+            _.map(_.range(0, 11), function (n) {
```

Comment:
Ah yeah, you're right. I always get confused, as I think the end is exclusive in Scala ...